### PR TITLE
new community page

### PR DESCRIPTION
**Pull request by mabis**
might be too soon in which case feel free to reject!

Signed-off-by: Marco Abis <marco.abis@gmail.com>

### CODE REVIEW

File: docs/introduction/1236-community.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+slug: /1236/community
+displayed_sidebar: "0.2"
+---
+
+# Join the Community
+
+Dagger is open source! Anyone is welcome to join the community and help shape the evolution of Dagger.
```
Comment (TomChv) [inline docs/introduction/1236-community.md:8]: `Open Source`

File: docs/introduction/1236-community.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Discord
+
+* [Dagger Official Discord](https://discord.gg/dagger-io)
+  * Rules:
+    * Be kind and respectful
+    * Follow [Discord's Community Guidelines](https://discord.com/new/guidelines)
```
Comment (TomChv) [inline docs/introduction/1236-community.md:15]: Should be better as sentences

**Review by TomChv — COMMENTED** [review]
Thanks for your contribution! That's a really good idea.

I see we should add some assets to make that page beautiful!

I let minors comments too

File: docs/introduction/1236-community.md (hunk 1 above)
Comment (mabis) [reply docs/introduction/1236-community.md:8]: I went for consistency and that sentence is taken from the bottom of https://dagger.io where "open source" is lower case. I'm fine wither way :-)

**Comment by mabis** [comment]
> it seems you have change from other's PR

indeed not sure what happened: I rebased my PR yesterday (from a different machine) and it brought with it changes from others's PRs

**Comment by TomChv** [comment]
What if you pull main and rebase your branch?

**Comment by mabis** [comment]
done, commit should be clean now

**Comment by shykes** [comment]
Thanks @mabis , this is a great contribution. We can merge as-is, and use it as a base for further improvements.

Could you 1) squash your commits into one, 2) prefix your commit message with `docs: ` ? Then we'll be ready to go.

**Review by shykes — APPROVED** [review]

**Comment by shykes** [comment]
Thank you!

**Pull request merged**